clang/include/clang/Basic/AttrDocs.td

```diff
@@ -415,6 +415,11 @@ def SYCLSimdDocs : Documentation {
     The compiler may decide to compile such functions using different optimization
     and code generation pipeline. Also, this attribute is used to distinguish
     ESIMD private globals from regular SYCL global variables.
+
+    When conforming to SYCL 1.2.1 specifications, when called from a device
```

Comment:
I am unsure about these `conforming to SYCL 1.2.1 specifications` and `conforming to SYCL 2020` since all these attributes have nothing to do with these specifications.